static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -133,11 +132,13 @@ class SpanTree extends Component<PropType> {
       return null;
     }
 
+    const isClickable = focusedSpanIds && showFilteredSpansMessage;
```

Comment:
While I was here, I added this to fix a bug where the 'out of view' messages were also clickable, which would give them a cursor hover and fire off the analytics event when they are clicked on